tests/integration/deploy/test_deploy_command.py

```diff
@@ -283,6 +283,7 @@ def test_deploy_with_s3_bucket_switch_region(self, template_file):
         template_path = self.test_data_path.joinpath(template_file)
 
         stack_name = "a" + str(uuid.uuid4()).replace("-", "")[:10]
+        self.stack_names.append(stack_name)
```

Comment:
Nice catch. Not needed now, but we should adjust this in the future to where we only delete the stack if the tests pass. Would help us have something inspect-able after the fact, instead of just errors in the output.